src/components/lyne-panel/lyne-panel.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Component,
+  h,
+  Prop
+} from '@stencil/core';
+import { InterfacePanelAttributes } from './lyne-panel.custom.d';
+
+@Component({
+  shadow: true,
+  styleUrl: 'lyne-panel.scss',
+  tag: 'lyne-panel'
+})
+
+export class LynePanel {
+
+  /** The text to show in the panel */
+  @Prop() public text!: string;
+
+  /** The text to use as button text */
+  @Prop() public buttonText!: string;
+
+  /** Id which is sent in the click event payload for the button*/
+  @Prop() public eventId?: string;
+
+  /** The tag to use for the text element */
+  @Prop() public tag?: InterfacePanelAttributes['tag'] = 'p';
+
+  public render(): JSX.Element {
+    const TAGNAME = this.tag;
+
+    return (
+      <div class='lyne-panel'>
+        <TAGNAME class='lyne-panel__text'>{this.text}</TAGNAME>
+
+        <lyne-button
```

Comment:
As mentioned in the chat, I think that the likelyhood that we actually have a button element in here seems to be rather low to me (even though we need the visual appearance of the button element for it). Looking back how the element was previously used, I suppose that in most cases `<div class="lyne-panel">` will be either an `<a>` element itself or will be packed as a `<div>` into another `<a>` element. If this should be able to become a link element and the white panel will also further exist, we need to think about how we can communicate the contents of the entire panel construction in one go. Means we eventually have to go back and add an aria-label attribute on the `<a>` element to let screenreaders speak out the contents at once.

Reply:
done via property on the button.